packages/turbo/ios/RNVisitableView.swift

```diff
@@ -14,14 +14,23 @@ let REFRESH_SCRIPT = "typeof Turbo.session.refresh === 'function'" +
 class RNVisitableView: UIView, RNSessionSubscriber {
   var id: UUID = UUID()
   @objc var sessionHandle: NSString? = nil
-  @objc var applicationNameForUserAgent: NSString? = nil
   @objc var url: NSString = "" {
     didSet {
       if(url != oldValue) {
         visit()
       }
     }
   }
+  @objc var applicationNameForUserAgent: NSString? = nil {
+    didSet {
+      sessionConfiguration.webViewConfiguration.applicationNameForUserAgent = applicationNameForUserAgent as? String
+    }
+  }
+  @objc var scrollEnabled: Bool = true {
+    didSet {
+      sessionConfiguration.isScrollEnabled = scrollEnabled
+    }
+  }
   @objc var pullToRefreshEnabled: Bool = true {
     didSet {
       controller!.visitableView.allowsPullToRefresh = pullToRefreshEnabled
```

Comment:
I feel like this could also be moved into the configurator provided we initialize the configurator with the `RNVisitableView` (which would allow us to access both session and controller).